README.md

```diff
@@ -50,20 +50,22 @@ Data (and functions related to that data) shared across multiple pages.
 ### `Page.`
 `Page.Writing.Rate.Main`, `Page.Writing.Rate.Update`, `Page.Writing.Rate.Model.Decoder`
 
-A page on the site, which has its own URL. These are not reusable, and implemented using a combination of types from `Data` and modules from `Nri`.
+A page on the site, which has its own URL. These are not reusable, and implemented using a combination of types from `Data` and modules from `Nri`. Comments for usage instructions aren't required, as code isn't intended to be reusable.
 
 The module name should follow the URL. Naming after the URL is subject to [How to Structure Modules for A Page](#how-to-structure-modules-for-a-page). The purpose of this convention is so when you have a URL, you can easily figure out where to find the module.
 
-Comments for usage instructions aren't required, as code isn't intended to be reusable.
+If a module ends with `Main`, everything between `Page` and `Main` **MUST** correspond to a URL.
 
 #### Examples
-- `Page.Learn.Quiz.MultiHighlighter.Main` corresponds to the URL `/learn/quiz/:id`. In particular, the `Learn.Quiz` part corresponds to the `/learn/quiz` part. The `MultiHighlighter.Main` part is subject to [How to Structure Modules for A Page](#how-to-structure-modules-for-a-page).
-- `Page.Admin.RelevantTerms.Main` corresponds to the URL `/admin/relevant_terms`. In particular, the `Admin.RelevantTerms` part corresponds to the `/admin/relevant_terms` part. The `Main` part is subject to [How to Structure Modules for A Page](#how-to-structure-modules-for-a-page).
+- `Page.Admin.RelevantTerms.Main` corresponds to the URL `/admin/relevant_terms`.
+- `Page.Learn.ChooseSourceMaterials.Main` corresponds to the URL `/learn/choose_source_materials`.
+- `Page.Preferences.Main` corresponds to the URL `/preferences`.
+- `Page.Teacher.Courses.Assignments.Main` corresponds to the URL `/teach/courses/:id/assignments`.
+    N.B. The `/:id` is dropped from the module name to avoid too much complexity.
 
 #### Non-examples
-- `Page.Teach.WritingCycles.Rate.Main` corresponds to no URL, so finding it from the URL will be hard.
-- `Page.Mastery.Main` corresponds to no URL, so finding it from the URL will be hard.
-
+- `Page.Mastery.Main` corresponds to no URL. We would expect `/mastery` to exist, but it doesn't. Finding the module from the URL will be hard.
+- `Page.Teach.WritingCycles.Rate.Main` corresponds to no URL. We would expect `/teach/writing_cycles/rate` or `/teach/writing_cycles/:id/rate` to exist, but neither do. Finding the module from the URL will be hard.
```

Comment:
Could we include an explanation of why we have non-examples in our codebase? I.e., explain this is older code and we've exempted it through the `TRIAGE` list in `lib/tasks/lint.rake`, but don't follow this non-example.

Reply:
That information feels like it belongs in our internal documentation. Will update it in https://github.com/NoRedInk/wiki/pull/859